squirrel.frontier/src/main/java/org/dice_research/squirrel/frontier/recrawling/OutDatedUriRetreiver.java

```diff
@@ -0,0 +1,18 @@
+package org.dice_research.squirrel.frontier.recrawling;
+
+import java.io.Closeable;
+import java.util.List;
+
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+
+public interface OutDatedUriRetreiver extends Closeable {
```

Comment:
1. Typo `Retreiver` should be `Retriever`

2.  Please add a Javadoc comment describing the meaning of the interface.